src/cookie.js

```diff
@@ -42,17 +42,25 @@ async function main() {
     let {host} = new URL(tab.url);
     let domain = document.querySelector("#domain");
     domain.textContent = host;
-
+    let report = document.getElementById("report");
     let site = await getSite(host);
-    if (site) {
-        document.body.className = "managed";
+    let nosite = document.getElementById("nosite");
+
+    if (tab.url.startsWith("about:") || tab.url.startsWith("file:") || tab.url == "") {
+        nosite.style.display = "initial";
```

Comment:
This is better, though I meant something like setting the `document.body.className = "nosite"` here and controlling the visibility of everything via css.

Though if you can't figure it out, I'll clean it up later.